### ISSUE TITLE
Audit the hamburger menu and mobile nav

### ISSUE THREAD

**Issue by TheOtterlord** [withastro/blog-tutorial-demo#11]
We've received a few pieces of feedback that the hamburger menu should be a button, in place of a div, in favour of better accessibility.

#### Steps:
1. Update the hamburger menu
2. Update the tutorial to match the new code

**Comment by sarah11918** [comment]
From our Feedback Fish triage, here are some things to check re: the Hamburger menu and mobile nav:

- styling choices (use a `<style>` tag instead of global styles to show scoped styling for the component?
- CSS display and expanded class
- should this be a button?
- verify all screen sizes for display:none
- are we generally using best styling/responsive practices for the Hamburger component and mobile nav?

**Comment by sarah11918** [comment]
We did this!
